src/gc/gcee.cpp

```diff
@@ -390,6 +390,7 @@ size_t GCHeap::GetNow()
     return hp->get_time_now();
 }
 
+#if defined(GC_PROFILING) //UNIXTODO: Enable this for FEATURE_EVENT_TRACE
 void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
 {
 #if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
```

Comment:
My suggestion was to remove the GC_PROFILING define from within the ProfScanRootsHelper function now that it is defined at an outer scope.

On Windows, everything gets compiled since both defines (gc profiling and event tracing) are defined. On Linux, gc profiling is not defined by event tracing is - thus, whenever GC profiling is enabled, the inner event tracing would be lit up too.